src/general/WorldGenerationSettings.cs

```diff
@@ -1,6 +1,175 @@
-﻿/// <summary>
+﻿using System;
+
+/// <summary>
 ///   Player configurable options for creating the game world
 /// </summary>
 public class WorldGenerationSettings
 {
+    /*
+    Static values for min/max
+    */
+
+    public const double MIN_MP_MULTIPLIER = 0.2;
+    public const double MAX_MP_MULTIPLIER = 2;
+    public const double MIN_AI_MUTATION_RATE = 0.5;
+    public const double MAX_AI_MUTATION_RATE = 5;
+    public const double MIN_COMPOUND_DENSITY = 0.2;
+    public const double MAX_COMPOUND_DENSITY = 2;
+    public const double MIN_PLAYER_DEATH_POPULATION_PENALTY = 1;
+    public const double MAX_PLAYER_DEATH_POPULATION_PENALTY = 5;
+    public const double MIN_GLUCOSE_DECAY = 0.3;
+    public const double MAX_GLUCOSE_DECAY = 0.95;
+    public const double MIN_OSMOREGULATION_MULTIPLIER = 0.2;
+    public const double MAX_OSMOREGULATION_MULTIPLIER = 2;
```

Comment:
I thought it might be cleaner to keep these constants with the rest of the world generation settings, since they're exclusively applicable here. But if the consensus says otherwise, I'll move them.

Reply:
I think maybe in this case the huge dump of difficulty option ranges should maybe be in the separate file. At least if they were moved to constants they'd need much more descriptive names, and I imagine (I haven't looked through this PR yet) that only the world generation really should use these. So the design is a bit clearer if other places in the code aren't able to access these and instead need to read the values that the current game has stored.

Reply:
I think they might make sense here (I haven't read the code for this PR yet), another place might be the new game GUI, since these look like to be the intervals the new game options need to be within. That way it would be super clear that only that new game GUI should use this (especially if these could be made private in there), and all other places in the code need to access the actual current game variables.